modules/simple_forms_api/app/services/simple_forms_api/s3/submission_builder.rb

```diff
@@ -0,0 +1,110 @@
+# frozen_string_literal: true
+
+module SimpleFormsApi
+  module S3
+    class SubmissionBuilder < Utils
```

Comment:
Should use of this class supersede my rake task where I do all this inline in `resubmit_forms_by_uuid.rake`? ie, call this class from my rake task and cut down the logic in the rake task by a lot

Reply:
Yeah, I think we could use this for that and possibly even in the UploadsController if we were so inclined. The idea is to make the submission generation as modular as possible in order to use it anywhere we see fit. I'd love your feedback or tradeoffs you can come up with for that idea.

Reply:
Ooh using it in UploadsController sounds interesting, too. Maybe this shouldn't live in `module S3` then? Seems like having a `SubmissionBuilder` service at the top level with our other `Pdf*` services makes sense. We can circle back and replace the logic in my rake task and UploadsController after this goes in though, I think.

Reply:
Yeah, I wanted to get your input about it before taking it out and also I think that can be another PR in the future once this solution is completed.